### DISCUSSION TITLE
List (Diagnostics) Update

### DISCUSSION THREAD

**Discussion by Golans** [gaelj/BlazorCodeMirror6#121] [Q&A]
Hi @gaelj 

Another question: 
Is there any option to force Lint's model updates (let's say after manual compilation)?

Golan

**Comment by gaelj** [comment]
Hi @Golans, sorry I didn't notice your message earlier. I'm not sure I understand exactly your question. What compilation are you referring to ?

The current implementation calls the linting callback `[Parameter] public Func<string, CancellationToken, Task<List<CodeMirrorDiagnostic>>>? LintDocument { get; set; }` each time the document is changed and after a small delay, to avoid linting on each character change. Implementation of the callback  is up to you.

**Comment by gaelj** [comment]
I just added a command that allows forcing a lint update, does that help?
